variables.tf

```diff
@@ -109,6 +115,12 @@ variable "administrative" {
   default     = false
 }
 
+variable "additional_project_globs" {
+  type        = set(string)
+  description = "Project globs is an optional list of paths to track stack changes of outside of the project root. Push policies are another alternative to track changes in additional paths."
```

Comment:
Aha -- meaning that if we change `config/github-repositories/terraform-spacelift-automation.yaml` then it will track those changes? How does that work... Does it include them in `affected_files` for that Stack somehow? I need to look into that more... that's very useful.

Reply:
Hm, [after reading the docs on this](https://docs.spacelift.io/concepts/stack/stack-settings#project-globs), I'm confused if it's mutually exclusive with push policies or not... 

> In the absence of push policies, any changes made to the project root and any paths specified by project globs will trigger Spacelift runs.

I'm interested how this ends up working out... keep me posted. Even if we can specify this and then access it in the push policy to allow ourselves to check Stack defined paths to check, that would be a nice win.

Reply:
Not sure exactly how it would work. Saw this on the docs and thought it might be helpful to add! We can experiment with it in `mp-infra`